src/zelos/config_gen.py

```diff
@@ -202,13 +206,15 @@ def generate_parser():
         "mount multiple files.",
     )
     group_fs.add_argument(
+        "-ev",
         "--env_vars",
-        action="append",
-        default=[],
+        metavar="KEY=VALUE",
+        default={},
         help="Emulated environment variables. ENV_VARS is a comma separated "
         "key value pair. Can be specified multiple times to set multiple "
         "environment variables. Format: '--env_vars FOO:bar --env_vars "
         "ZERO:point'.",
```

Comment:
Does the metavar imply that the format requires an `=` inbetween the key and the value? If so, we should update the help text